projects/core/styles/theme/palette.less

```diff
@@ -1,4 +1,5 @@
-[tuiTheme='dark'] {
+[tuiTheme='dark'],
+&:root:has(tui-root[tuiTheme='dark']) {
```

Comment:
Please only add this to the demo:
1) palette should not depend on where you put `tuiTheme` attribute
2) if you write it like that it will be broken for browsers not supporting `:has`, since they will skip entire rule if one of the selectors with `,` is not supported

Reply:
fixed